packages/app-aco/src/contexts/records.tsx

```diff
@@ -73,19 +76,44 @@ export const SearchRecordsProvider = ({ children }: Props) => {
         records,
         loading,
         meta,
-        async listRecords(type: string, folderId: string, limit = 10, after?: string) {
-            if (!folderId) {
-                throw new Error("`folderId` is mandatory");
+        async listRecords(
+            type?: string,
+            folderId?: string,
+            limit = 20,
+            after?: string,
+            sorting?: string[]
+        ) {
+            if (!folderId || !type) {
+                throw new Error("`folderId` and `type` are mandatory");
```

Comment:
We need to use `useRecords` to import the new method `getRecord` (useful for sinking records after page actions).

Now, we use `useRecords` without `folderId` and `type`...so we don't need to propagate these information to multiple components.

In this line, we check that `folderId` and `type` are passed to `listRecords`...both are required for this method.

Reply:
If so, then please add a comment describing the various scenarios of calling this method. Because the method type shows that both parameters are optional.